fud/fud/stages/vivado/extract.py

```diff
@@ -69,6 +70,27 @@ def futil_extract(directory):
         f8_muxes = to_int(find_row(slice_logic, "Site Type", "F8 Muxes")["Used"])
         f9_muxes = to_int(find_row(slice_logic, "Site Type", "F9 Muxes")["Used"])
 
+        resourceInfo.update({
+            "lut": to_int(find_row(slice_logic, "Site Type", "CLB LUTs")["Used"]),
+            "dsp": to_int(find_row(dsp_table, "Site Type", "DSPs")["Used"]),
+            "meet_timing": int(meet_timing),
+            "registers": rtl_component_extract(directory, "Registers"),
+            "muxes": rtl_component_extract(directory, "Muxes"),
+            "clb_registers": clb_reg,
+            "carry8": carry8,
+            "f7_muxes": f7_muxes,
+            "f8_muxes": f8_muxes,
+            "f9_muxes": f9_muxes,
+            "clb": clb_lut + clb_reg + carry8 + f7_muxes + f8_muxes + f9_muxes,
+        })
+
+    except Exception as e:
+        import traceback
```

Comment:
Can you place the import at the top and add it as a dependency to the pyproject.toml file so that it is correctly installed when `fud `is installed?

Reply:
I'll move the imports; adding to pyproject.toml shouldn't be necessary since traceback is part of the standard library.